components/job-orchestration/job_orchestration/scheduler/compress/compression_scheduler.py

```diff
@@ -209,12 +209,14 @@ def search_and_schedule_new_tasks(db_conn, db_cursor, clp_metadata_db_connection
 
         if len(tasks) == 0:
             logger.warning(f"No tasks were created for job {job_id}")
+            update_time = datetime.datetime.now()
```

Comment:
Would it be better if we add this logic into `update_compression_job_metadata` so whenever we update the job, the `update_time` can be updated?

Reply:
I agree, it would be cleaner to do so.